src/_cffi_src/build_openssl.py

```diff
@@ -37,7 +37,11 @@ def _osx_libraries(build_static):
         return ["ssl", "crypto"]
 
 
-_OSX_PRE_INCLUDE = """
+_PRE_INCLUDE = """
+#include <openssl/e_os2.h>
```

Comment:
:+1:

On Mon, Jan 18, 2016 at 10:11 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/_cffi_src/build_openssl.py
> <https://github.com/pyca/cryptography/pull/2678#discussion_r50068570>:
>
> > @@ -37,7 +37,11 @@ def _osx_libraries(build_static):
> >          return ["ssl", "crypto"]
> >
> >
> > -_OSX_PRE_INCLUDE = """
> > +_PRE_INCLUDE = """
> > +#include <openssl/e_os2.h>
>
> It works out okay.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2678/files#r50068570>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084